org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -160,40 +163,41 @@ public static String getFieldWithUpperCaseFirstLetter(final String fieldName) {
 
 
 	protected Map<String, U> getMap(String queueName) throws EventException {
-		
+
 		final List<U> queue = getQueue(queueName, null);
 		if (queue==null || queue.isEmpty()) return null;
 		final HashMap<String, U> id = new HashMap<>(queue.size());
 		for (U u : queue) id.put(u.getUniqueId(), u);
 		return id;
 	}
 
+	@SuppressWarnings({"squid:S1192", "squid:S3776"})
 	@Override
 	public void cleanQueue(String queueName) throws EventException {
 
 		try {
 			QueueConnection qCon = null;
-	
+
 			try {
 				QueueConnectionFactory connectionFactory = (QueueConnectionFactory)service.createConnectionFactory(uri);
-				qCon  = connectionFactory.createQueueConnection(); 
+				qCon  = connectionFactory.createQueueConnection();
 				QueueSession    qSes  = qCon.createQueueSession(false, Session.AUTO_ACKNOWLEDGE);
 				Queue queue   = qSes.createQueue(queueName);
 				qCon.start();
-	
+
 				QueueBrowser qb = qSes.createBrowser(queue);
-	
+
 				@SuppressWarnings("rawtypes")
 				Enumeration  e  = qb.getEnumeration();
-	
+
 				Map<String, StatusBean> failIds = new LinkedHashMap<String, StatusBean>(7);
 				List<String>          removeIds = new ArrayList<String>(7);
 				while(e.hasMoreElements()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)